src/openklant/conf/base.py

```diff
@@ -1,448 +1,40 @@
-import os
-import warnings
-
-from django.urls import reverse_lazy
-
-import sentry_sdk
-from corsheaders.defaults import default_headers as default_cors_headers
-from log_outgoing_requests.formatters import HttpFormatter
+from open_api_framework.conf.base import *  # noqa
 
 from .api import *  # noqa
-from .includes.environ import config, get_sentry_integrations
-
-# Build paths inside the project, so further paths can be defined relative to
-# the code root.
-
-DJANGO_PROJECT_DIR = os.path.abspath(
-    os.path.join(os.path.dirname(__file__), os.path.pardir)
-)
-BASE_DIR = os.path.abspath(
-    os.path.join(DJANGO_PROJECT_DIR, os.path.pardir, os.path.pardir)
-)
-
-#
-# Core Django settings
-#
-SITE_ID = config("SITE_ID", default=1)
-
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config("SECRET_KEY")
-
-# NEVER run with DEBUG=True in production-like environments
-DEBUG = config("DEBUG", default=False)
-
-# = domains we're running on
-ALLOWED_HOSTS = config("ALLOWED_HOSTS", default="", split=True)
-USE_X_FORWARDED_HOST = config("USE_X_FORWARDED_HOST", default=False)
-
-IS_HTTPS = config("IS_HTTPS", default=not DEBUG)
-
-# Internationalization
-# https://docs.djangoproject.com/en/2.0/topics/i18n/
-
-LANGUAGE_CODE = "nl"
-
-TIME_ZONE = "UTC"
-
-USE_I18N = True
-
-USE_L10N = True
-
-USE_TZ = True
-
-USE_THOUSAND_SEPARATOR = True
-
-#
-# DATABASE and CACHING setup
-#
-DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.postgresql",
-        "NAME": config("DB_NAME", "openklant"),
-        "USER": config("DB_USER", "openklant"),
-        "PASSWORD": config("DB_PASSWORD", "openklant"),
-        "HOST": config("DB_HOST", "localhost"),
-        "PORT": config("DB_PORT", 5432),
-    }
-}
-
-# keep the current schema for now and deal with migrating to BigAutoField later, see
-# https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-DEFAULT_AUTO_FIELD
-DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
-
-CACHES = {
-    "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_DEFAULT', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-    "axes": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_AXES', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-    "oidc": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_OIDC', 'localhost:6379/0')}",
```

Comment:
Another place where `CACHE_OIDC` is mentioned but is different in open-api-framework